src/Stratis.Bitcoin.IntegrationTests/Mempool/MempoolRelaySpecification.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Stratis.Bitcoin.IntegrationTests.TestFramework;
+using Xunit;
+// ReSharper disable ArrangeThisQualifier
+
+namespace Stratis.Bitcoin.IntegrationTests.Mempool
+{
+    public partial class MempoolRelaySpecification : BddSpecification
+    {
+        [Fact]
+        public void create_transaction_and_broadcast_to_whitelisted_nodes_SHOULD_get_propagated_to_third_peer()
+        {
+            this.Given(nodeA_nodeB_and_nodeC);
```

Comment:
removing all these "this." would be cleaner in these bdd tests

Reply:
ah force of habit, but then again I like the `this.`

Reply:
I like it too, in normal production code. I just think when it gets this close to a clean line of English, it would be a shame to have `this.`